### PR TITLE
Function for unique naming

### PR DESCRIPTION
**Pull request by ebenahar**

### CODE REVIEW

**Comment by ebenahar** [comment]
Verified. Executed this simple test that prints the created name:

```python
import logging
from tests import helpers

log = logging.getLogger(__name__)


class Test1:
    def test_create_name(self):
        name = helpers.create_unique_object_name('test', 'project')
        log.info(f"{name}")
```

tests/manage/test_1.py::Test1::test_create_name 
-------------------------------------------------------------------------------------------------- live log call --------------------------------------------------------------------------------------------------
12:10:25 - MainThread - tests.manage.test_1 - INFO - project_test_2912102521
PASSED

File: tests/helpers.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Helper functions file for OCS QE
+"""
+import datetime
+
+
+def create_unique_object_name(object_description, object_type):
+    """
+    Creates a unique object name by using the object_description
+    and object_type, as well as the current date/time string.
+    This can be used for any objects such as VMs, disks, clusters etc.
```
Comment (petr-balogh) [inline tests/helpers.py]: VMs, disks, cluster etc -> Change to something OCS related ;)

Comment (ebenahar) [reply tests/helpers.py]: Done :)

File: tests/helpers.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+"""
+Helper functions file for OCS QE
+"""
+import datetime
+
+
+def create_unique_object_name(object_description, object_type):
```
Comment (RazTamir) [inline tests/helpers.py]: I will change it to resource_description and resource_type

Comment (PrasadDesala) [reply tests/helpers.py]: +1
Also, I think we need to change the function name the same way as the args name. Something to..
"create_unique_resource_name"

**Comment by PrasadDesala** [comment]
Also just noticed that, this file is placed in tests/helpers.py which may not be a perfect place given that it is not a test case.
**Please ignore this comment if you have plans to move it at a later stage.

**Comment by ebenahar** [comment]
@PrasadDesala I think this is the correct location since this function will be used for resources created as part of the tests

**Comment by keesturam** [comment]
Looks good to me.

**Comment by mbukatov** [comment]
> @PrasadDesala I think this is the correct location since this function will be used for resources created as part of the tests

I disagree. We should maintain some separation between test code and utility modules. Moreover if someone decides to create subdirectories in the tests directory, there is no reliable way to import the module.

**Comment by ebenahar** [comment]
@PrasadDesala @mbukatov Ok, I get your point.
How about utility/utils.py?

**Pull request merged**